spec/project-spec.js

```diff
@@ -1064,9 +1064,14 @@ describe('Project', () => {
           for (let event of events) {
             remaining.delete(event.path);
           }
+          clearTimeout(expireTimeoutId);
           if (remaining.size === 0) {
-            clearTimeout(expireTimeoutId);
             resolve();
+          } else {
+            console.error('Paths not seen:', remaining);
+            reject(
+              new Error('All events were processed but not all paths were seen.')
+            );
```

Comment:
```suggestion
            reject(
              new Error(`All events were processed but not all paths were seen. Paths not seen: ${remaining}`)
            );
```

Reply:
Agree with this change but since `remaining` is a Set, it will be stringified to `[object Set]`
![image](https://user-images.githubusercontent.com/58114641/176970285-62679911-4e5c-4994-8675-4042b7396dfa.png)

```suggestion
            const remainingStr = [...remaining].join('\n')
            reject(
              new Error(
                `All events were processed but not all paths were seen. Paths not seen: ${remainingStr}`
              )
            );
```

Reply:
Yes, you can use something like `[...remaining]`